lib/screens/signupCallAction.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+import 'package:memory_lamp/defaults/ml_text.dart';
+import 'package:memory_lamp/helpers/size_mq.dart';
+import 'package:memory_lamp/screens/signup.dart';
+
+class SignUpHere extends StatelessWidget {
```

Comment:
and pa-alis nito sir sa "screens" directory
and palagay na lang sa loob ng SignUpScreen as a private widget since hindi naman to gagamitin sa ibang screens mas maganda kung hindi sya public

Reply:
noted po